target/linux/ar71xx/image/senao.mk

```diff
@@ -0,0 +1,30 @@
+define Build/senao-factory-image
+	mkdir -p $(KDIR_TMP)/$(1)
+	touch $(KDIR_TMP)/$(1)/FWINFO-OpenWrt-$(REVISION)-$(1)
+	# $(CP) ./$(1)/* $(KDIR_TMP)/$(1)
+	dd if=$(KDIR)/$(1)-kernel.bin \
+		of=$(KDIR_TMP)/$(1)/openwrt-senao-$(1)-uImage-lzma.bin bs=64k conv=sync
+	dd if=$(KDIR)/root.squashfs \
+		of=$(KDIR_TMP)/$(1)/openwrt-senao-$(1)-root.squashfs bs=64k conv=sync
+	( \
+		cd $(KDIR_TMP)/$(1);  \
+		$(TAR) -cz -f $(call factoryname,squashfs,$(1)) * \
+	)
+	rm -rf $(KDIR_TMP)/$(1)
+endef
+
+define Device/ens202ext
```

Comment:
Using sysupgrade.tar here is wrong. sysupgrade.tar is for used for NAND flash chips with ubi. There is no need to have a dedicated kernel and rootfs partition. You can combine them to a firmware partition and use the dynamic firmware splitter in LEDE.

Would you please try the following instead: